### PR TITLE
Jira 22

### PR DESCRIPTION
**Pull request by SherryGupta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Following link will give details infirmation of all cdk actions.
 https://docs.aws.amazon.com/cdk/latest/guide/tools.html
 
 
-Following templates are created for this pipeline : - 
- 1- create-user-role.js will create a aws user  with full admin acess to CodeCommmit repo
+Following templates are created for this pipline : - 
+ 1- create-user-role.js will create a aws user  with full admin access to CodeCommmit repo
```
Comment (pontusvision) [inline README.md]: change the file name to reflect the camel case

File: README.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Following link will give details infirmation of all cdk actions.
 https://docs.aws.amazon.com/cdk/latest/guide/tools.html
 
 
-Following templates are created for this pipeline : - 
- 1- create-user-role.js will create a aws user  with full admin acess to CodeCommmit repo
+Following templates are created for this pipline : -
```
Comment (pontusvision) [inline README.md]: pipeline is mispelled

File: README.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ Following link will give details infirmation of all cdk actions.
...
-Following templates are created for this pipeline : - 
- 1- create-user-role.js will create a aws user  with full admin acess to CodeCommmit repo
+Following templates are created for this pipline : - 
+ 1- create-user-role.js will create a aws user  with full admin access to CodeCommmit repo
 This user will be used to manually push all code in the repo.
+
+2-CreateInfraPipelineStack.ts
+    This will create repo to store infrastructure bootstrap scripts and then create pipeline  to build/deploy all infrastructure code.
+    
+3.CreateCodePipelineStack
```
Comment (pontusvision) [inline README.md]: perhaps rename this to CreateAppPipeline to avoid conflicts with CreateCodePipeline?

File: lib/CreateCodePipelineStack.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+export interface PipelineStackProps extends StackProps {
+    readonly lambdaCode: lambda.CfnParametersCode;
+}
+
+
+export class CreateCodePipelineStack extends Stack {
+    constructor(app: App, id: string, props: PipelineStackProps) {
+        super(app, id, props);
+
+        const codeRepo = new codecommit.Repository(this, 'AppRepo', {
```
Comment (pontusvision) [inline lib/CreateCodePipelineStack.ts]: Consider parameterising the appRepo name

File: test/CreateCodePipelineStack.test.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import lambda = require('@aws-cdk/aws-lambda');
+
+export class LambdaStack extends Stack {
+    public readonly lambdaCode: lambda.CfnParametersCode;
+
+    constructor(app: App, id: string, props?: StackProps) {
+        super(app, id, props);
+        this.lambdaCode = lambda.Code.cfnParameters();
+        const func = new lambda.Function(this, 'Lambda', {
+            code: this.lambdaCode,
```
Comment (pontusvision) [inline test/CreateCodePipelineStack.test.ts]: Consider using an inline function (see https://stackoverflow.com/questions/55223496/how-to-dynamically-create-an-inline-lambda-with-aws-sdk-for-javascript)

File: test/CreateCodePipelineStack.test.ts [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+
+test('code repository created', () => {
+    expect(myConstruct).to(haveResource('AWS::CodeCommit::Repository', {
+            "RepositoryName": "AppRepo"
+        }, ResourcePart.Properties)
+    );
+
+});
+
```
Comment (pontusvision) [inline test/CreateCodePipelineStack.test.ts]: Consider adding a test case to look at the lambda code that we created above

File: README.md (hunk 1 above)
Comment (SherryGupta) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (SherryGupta) [reply README.md]: done

File: lib/CreateCodePipelineStack.ts (hunk 4 above)
Comment (SherryGupta) [reply lib/CreateCodePipelineStack.ts]: done

File: README.md (hunk 3 above)
Comment (SherryGupta) [reply README.md]: done

File: lib/CreateCodePipelineStack.ts (hunk 4 above)
Comment (SherryGupta) [reply lib/CreateCodePipelineStack.ts]: done

File: test/CreateCodePipelineStack.test.ts (hunk 5 above)
Comment (SherryGupta) [reply test/CreateCodePipelineStack.test.ts]: I tried to use inline function but its not giving any benefit as its not reducing number of lines of codes.

File: test/CreateCodePipelineStack.test.ts (hunk 6 above)
Comment (SherryGupta) [reply test/CreateCodePipelineStack.test.ts]: done

**Pull request merged**